internal/build/boil_steps.go

```diff
@@ -0,0 +1,31 @@
+package build
+
+import (
+	"github.com/windmilleng/tilt/internal/model"
+)
+
+type BoiledStep struct {
+	cmd         model.Cmd
+	pathMapping pathMapping
+}
+
+func BoilSteps(steps []model.Step, pathMappings []pathMapping) ([]BoiledStep, error) {
+	res := []BoiledStep{}
+	for _, step := range steps {
+		if step.Trigger == nil {
+			res = append(res, BoiledStep{cmd: step.Cmd})
+			continue
+		}
+		for _, pm := range pathMappings {
+			matches, err := step.Trigger.Matches(pm.LocalPath, false)
```

Comment:
it seems like you could pass in the LocalPath as FilesChanged(), like you had before? you don't use the rest of the pathMapping